CONTRIBUTING.md

```diff
@@ -30,9 +55,12 @@
     from our issue tracker.  The body describes the behavior without the patch,
     why this is a problem, and how the patch fixes the problem when applied.
 ````
-
 * During the time that you are working on your patch the master Beaker branch may have changed - you'll want to [rebase](http://git-scm.com/book/en/Git-Branching-Rebasing) on top of [Beaker's](https://github.com/puppetlabs/beaker) master branch before you submit your PR.  A successful rebase ensures that your PR will cleanly merge into Beaker.
 
+#### Courtesy
+
+Please do not introduce personal ignores into the `.gitignore`, such as IDE configurations, editor version files, or personal testing detritus. You may find it valuable to add the first two to [a global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), and the third to [a repository-level ignore](https://help.github.com/articles/ignoring-files/#explicit-repository-excludes).
```

Comment:
let's not call what people who are trying to help us make 'detritus'. Something a little more neutral... artifacts?

Reply:
No malice intended; "artifact" is definitely better.